### DISCUSSION TITLE
英文引号编译后显示为中文引号

### DISCUSSION THREAD

**Discussion by Becmamba** [sjtug/SJTUThesis#666] [提问] [answered]
请问输入英文引号编译后显示为中文引号是怎么回事，单双引号都是。overleaf

**Comment by skyzh** [comment, marked as answer]
中文衬线字体里的英文引号就是这个字形。如果需要笔直的引号，请使用 `\textquotedbl`

<img width="571" alt="image" src="https://user-images.githubusercontent.com/4198311/119990626-21d16680-bffb-11eb-8652-59a955a07d1c.png">

**Reply by skyzh** [reply]
`\textquotedbl` 的效果如下。

<img width="571" alt="image" src="https://user-images.githubusercontent.com/4198311/119990836-665d0200-bffb-11eb-87d6-d077ae41554b.png">

**Reply by Becmamba** [reply]
收到，谢谢，双引号解决了，那么笔直的单引号该怎么写呢？没有搜到

**Reply by zepinglee** [reply]
直的单/双引号只出现在代码中，正文中一般不出现。

**Reply by skyzh** [reply]
> 收到，谢谢，双引号解决了，那么笔直的单引号该怎么写呢？没有搜到

参考 https://tex.stackexchange.com/questions/52529/straight-quotes ，可以使用数学环境的单引号。如 zeping 所说，正文中一般不出现直的引号。

**Comment by zepinglee** [comment]
参考 https://github.com/CTeX-org/ctex-kit/issues/389 。
